charts/crossplane-azure-upbound/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: crossplane-azure-upbound
+description: A Helm chart for the Azure Provider for Crossplane
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 2.0.2
```

Comment:
any reason you didn't make version and appVersion 1.0.0?

Reply:
This will avoid people wondering where is the previous version 1.x, if the first release is 2.x

Reply:
Done 😸 I had the same thought by the way. I changed the version and appVersion to 1.0.0